gui-apps/wf-recorder/wf-recorder-0.2.1.ebuild

```diff
@@ -7,20 +7,32 @@ inherit meson
 
 DESCRIPTION="Screen recorder for wlroots-based compositors"
 HOMEPAGE="https://github.com/ammen99/wf-recorder"
-SRC_URI="https://github.com/ammen99/${PN}/releases/download/v${PV}/${P}.tar.xz"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/ammen99/wf-recorder.git"
+else
+	SRC_URI="https://github.com/ammen99/wf-recorder/releases/download/v${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
+fi
 
 LICENSE="MIT"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
 IUSE="+man opencl"
 
-DEPEND="dev-libs/wayland
-	dev-libs/wayland-protocols
+DEPEND="
+	dev-libs/wayland
 	media-sound/pulseaudio
-	media-video/ffmpeg
-	opencl? ( virtual/opencl )"
+	media-video/ffmpeg[opencl?,pulseaudio,x264]
+	opencl? ( virtual/opencl )
+"
 RDEPEND="${DEPEND}"
-BDEPEND="man? ( app-text/scdoc )"
+BDEPEND="
+	${DEPEND}
```

Comment:
Can you tell me why you think this is needed? `BDEPEND` was designed to be a splitted off from `DEPEND`. There are cases where this makes sense, but I don't believe this to be one.

Reply:
removed